client/state/inline-help/selectors/get-contextual-help-results.js

```diff
@@ -20,7 +20,7 @@ import 'calypso/state/inline-help/init';
  */
 export default flow(
 	getLastRouteAction,
-	( x ) => x.path,
+	( x ) => x?.path,
```

Comment:
This works but could you let me know when this would be null?

Also, a very good janitorial task in the future is to implement a test suite for this selector. It's terrible that such an important selector doesn't have any test coverage.

Reply:
I am actually not sure what this part of the code does.  Just included a null check here. But I would be happy to dig a bit deeper and understand this further. Just on an initial glance seems like it is related to action logs. Do you think It's better to inestigate this further before proceeding with this commit, or can I take a look at this in a later commit? For now I think we can leave the null check.